andrea/js/app.js

```diff
@@ -0,0 +1,96 @@
+// buttons //
+let signUpButton = document.querySelector("#navbar-sign");
+let logInButton = document.querySelector("#navbar-login");
+
+let createButton = document.querySelector(".create-btn")
+
+let welcomeView = document.querySelector(".welcome-view")
+
+// form //
+let form = document.querySelector(".register-form");
+let registerTitle = document.querySelector(".register-title");
+let logInTitle = document.querySelector(".login-title");
+
+let registerFields = Array.from(document.querySelectorAll(".register-fields"));
+
+
+let logInForm = document.querySelector(".login-form");
+let formAlternative = document.querySelector(".form-alternative");
+
+let registerHide = [welcomeView, signUpButton, logInButton];
+let logInHide = [registerTitle, signUpButton, logInButton, createButton, welcomeView, formAlternative];
+logInHide = logInHide.concat(registerFields);
```

Comment:
Can you check if this works? `const logInElements = [registerTitle, signUpButton, logInButton, createButton, welcomeView, formAlternative, ...registerFields]`. If so, line 22 can be deleted